jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/factory/InputDialogGenerationStrategy.java

```diff
@@ -173,6 +179,24 @@ protected Component createBooleanField() {
         return uiComponents.create(JmixCheckbox.class);
     }
 
+    protected Component createFileStorageUploadField() {
+        FileStorageUploadField fileStorageUploadField = uiComponents.create(FileStorageUploadField.class);
+
+        fileStorageUploadField.setFileNameVisible(true);
```

Comment:
It's better to set `setWidthFull()` as for other components even though upload field file name component ignores it for now